libs/config.js

```diff
@@ -50,7 +54,8 @@ const domainToEnv = {
   'cozy.wtf': 'dev',
   'cozy.works': 'int',
   'cozy.rocks': 'stg',
-  'mycozy.cloud': 'prod'
+  'mycozy.cloud': 'prod',
+  'tools:8080': 'local'
```

Comment:
Isn't the local domain should be `cozy.tools:8080` instead ?

Reply:
normal cozy: toto.mycozy.cloud
local cozy: cozy.tools:8080

The the "standard" way of taking the domain is `fullDomain.split('.').slice(0).join('.')` so cozy.tools:8080 becomes "tools:8080".  Maybe cozy-url could help here, I agree that this is not optimal :)